model/wd2/index-respec.html

```diff
@@ -189,26 +189,32 @@
 
 <dl>
 
-  <dt>Resource</dt>
-  <dd>An item of interest that may be identified by a URI (a Web Resource), or may not have an identifier (a Blank Node).</dd>
+  <dt><dfn data-lt="Resource|Resources">Resource</dfn></dt>
+  <dd>An item of interest that may be identified by a URI (although not necessarily).</dd>
 
-  <dt>Web Resource</dt>
-  <dd>An item of interest that is identified by a URI, as described in the Web Architecture [[webarch]].  Web Resources may be dereferencable from their URI.</dd>
+  <dt><dfn data-lt="Web Resource|Web Resources">Web Resource</dfn></dt>
+  <dd>An item of interest, i.e., a <a>Resource</a> that is identified by a URI, as described in the Web Architecture [[webarch]].  Web Resources may be dereferencable from their URI.</dd>
 
-  <dt>External Web Resource</dt>
-  <dd>A Web Resource which is not part of the representation of the Annotation, such as a web page, image or video. External Web Resources are dereferencable from their URI.  Conversely a Specific Resource is not an External Web Resource, as its state is entirely captured within the Annotation's representation.</dd>
+  <dt><dfn data-lt="External Web Resource|External Web Resources">External Web Resource</dfn></dt>
+  <dd>A <a>Web Resource</a> which is not part of the representation of the Annotation, such as a web page, image or video. External Web Resources are dereferencable from their URI.
 
-  <dt>Property</dt>
```

Comment:
I don't think you meant to delete the Property definition? Not sure about adding SpecificResource here, especially given the discussion around #110?  Will copy the details out (as they need to go into the template rather than index-respec directly